backends/conrod_wgpu/src/lib.rs

```diff
@@ -563,11 +567,13 @@ fn bind_group_layout(
             component_type: GLYPH_TEX_COMPONENT_TY,
             dimension: wgpu::TextureViewDimension::D2,
         },
+        count: None,
     };
     let sampler_binding = wgpu::BindGroupLayoutEntry {
         binding: 1,
         visibility: wgpu::ShaderStage::FRAGMENT,
-        ty: wgpu::BindingType::Sampler { comparison: false },
+        ty: wgpu::BindingType::Sampler { comparison: true },
```

Comment:
This change is needed because `sampler_desc()` returns a sampler that does comparison (`compare: Some(wgpu::CompareFunction::Always)`)